configuration/restdocs.md

```diff
@@ -87,6 +87,18 @@ To authenticate with an API token, **apaddpend `-u '{API_TOKEN}:'` to the comman
 
 You can manage all access tokens in your profile settings in the Main UI.
 
+### Disable authentication
+
+It is possible to disable authentication.
+Stop openhab and add this part to ```userdata/etc/org.apache.karaf.features.xml``` before ```</featuresProcessing>```:
```

Comment:
We should consider using `$OPENHAB_USERDATA` as term placeholder here like done in other articles.

```
$OPENHAB_USERDATA/etc/org.apache.karaf.features.xml
```